### PR TITLE
Firefox 75: Add loading attribute to <img> element

### PR DESCRIPTION
**Pull request by a2sheppy**
* Added the `loading` attribute to the [HTMLImageElement](https://wiki.developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement) page
* Mention `loading` on [Web performance resources](https://wiki.developer.mozilla.org/en-US/docs/Learn/Performance/Web_Performance_Basics)
* Add info to [Lazy loading](https://wiki.developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading)
* Add [Use lazy loading for images](https://wiki.developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Author_fast-loading_HTML_pages#Use_lazy_loading_for_images) to "Tips for authoring fast-loading HTML pages]
* Updated [Firefox 75 for developers](https://wiki.developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/75)

Sources:
  * https://bugzilla.mozilla.org/show_bug.cgi?id=1542784
  * https://bugzilla.mozilla.org/show_bug.cgi?id=1613611
  * https://html.spec.whatwg.org/#attr-img-loading

### CODE REVIEW

File: api/HTMLImageElement.json [hunk 1]
```diff
@@ -669,6 +669,56 @@
...
+            },
+            "edge": {
+              "version_added": "79"
+            },
+            "firefox": {
+              "version_added": "75"
+            },
+            "firefox_android": {
+              "version_added": false,
+              "notes": "See <a href='https://bugzil.la/1542784'>bug 1542784</a>"
```
Comment (Elchi3) [inline api/HTMLImageElement.json]: This refers to a bug that is resolved fixed and I don't see information about Firefox Android there right away. Can you clarify why this bug reference is needed or is it the wrong bug?

Comment (ddbeck) [reply api/HTMLImageElement.json:689]: @a2sheppy This PR is still blocked on this question, if you get a chance to take a look soon. Thank you!

Comment (Elchi3) [reply api/HTMLImageElement.json:689]: @a2sheppy ping

Comment (a2sheppy) [reply api/HTMLImageElement.json]: Change submitted. Sorry about that.

**Pull request merged**